ci/test

```diff
@@ -110,12 +110,11 @@ rspec() {
 rspec_audit() {
   : DOC - Runs RSpecs for the Audit engine
   
-  # Start Conjur with the audit database
-  docker-compose up --no-deps -d audit pg
+  # Start Conjur with the postgres database
```

Comment:
They (audit+pg) are both "postgres" databases, so saying start conjur with the postgres database is a bit confusing... Maybe it will be clearer if we address to pg as "main" database

Reply:
In this case we are starting empty postgres database and not main database.